documentation/modules/ROOT/pages/configuration/filtering.adoc

```diff
@@ -9,26 +9,33 @@
 
 toc::[]
 
-[NOTE]
-====
-This SMT is currently in incubating state, i.e. exact semantics, configuration options etc. may change in future revisions, based on the feedback we receive. Please let us know if you encounter any problems while using this filter transformation.
-====
+By default, {prodname} delivers every data change event that it receives to the Kafka broker.
+However, in many cases, you might be interested in only a subset of the events emitted by the producer. 
+To enable you to process only the records that are relevant to you, {prodname} provides the _filter_ simple message transform (SMT).
```

Comment:
```suggestion
To enable you to process only the records that are relevant to you, {prodname} provides the `Filter` simple message transform (SMT).
```

Reply:
:thinking: Hmm. I had used that convention originally, but following a conversation with @TovaCohen, I adopted the lowercasing without the tick marks. I thought that usage was based on feedback that you and @gunnarmorling provided on the outbox event router doc. Not sure if I misunderstood. 

In any case, I agree that, especially on first mention, it might be good to highlight the term somehow. Are you suggesting that, or would you prefer to see `Filter` used consistently throughout?

Reply:
The issue with the outbox one was the usage was `outbox.EventRouter` more making a reference to package naming per-se.  If you guys agreed to use lowercase without the ticks, that's fine.  Just so long as we're consistent, I think that's the most important take-away.

Reply:
So there's two different represents:

* The "semantical name", e.g. "outbox event router", "content-based router", "filter", etc.
* The actual class name, e.g. `io.debezium.transforms.outbox.EventRouter`, etc.

From a reading flow perspective I think the semantical name should be used most of the times in docs, e.g. when describing its purpose etc. The class name should be used when pointing out how this is actually used. I like how that's done for outbox now, with the [Basic configuration](https://debezium.io/documentation/reference/configuration/outbox-event-router.html#_basic_configuration) section being the one that introduces the class name in one single place. Would be nice to have this done similarly here, too. Btw. this should be _before_ the actual options for outbox, i.e. "Basic configuration" should be the first under "Configuration". Could you do this as part of this PR? Thx!

Reply:
I made this change as part of the latest commit.